backend/app/assets/javascripts/spree/backend/format_money.js.erb

```diff
@@ -0,0 +1,27 @@
+//= require spree/backend/translation
+//= require solidus_admin/accounting
+
+Spree.currencyInfo = <%=
+Money::Currency.all.map { |c|
+  format =
+    if c.symbol == "" || c.symbol_first
+      "%s%v"
+    else
+      "%v %s"
+    end
+  [c.id.to_s.upcase, [
+    c.symbol || "¤",
+    c.exponent,
+    format
+  ]]
+}.to_h.to_json
+%>;
```

Comment:
We can't store this information like this as Sprockets won't pick up changes to `Money` settings while compiling new assets. Means, every time you change a `Money` setting you have to delete the assets on the server and force recompilation. 

To avoid this, we need to store this information in a html.erb file, like we already do for the JS translations data (`backend/app/views/spree/admin/shared/_translations.html.erb`)